src/components/relatedLinks/template.tsx

```diff
@@ -1,63 +1,66 @@
 import { isEmpty } from 'lodash'
-import { FormattedRelatedLinks } from '@/components/relatedLinks/formatted-type'
+import { ListOfLinkTeasers } from '@/components/listOfLinkTeasers/formatted-type'
 import { recordEvent } from '@/lib/analytics/recordEvent'
+import { LinkTeaser } from '../linkTeaser/formatted-type'
 
+/**
+ * Right now this is only used on the VAMC System and VAMC Facility pages as their own
+ * special way of displaying related links. Note that link summaries are not shown.
+ */
 export const RelatedLinks = ({
-  links,
-  sectionTitle,
-}: FormattedRelatedLinks): React.JSX.Element => {
-  if (isEmpty(links)) {
+  linkTeasers,
+  title,
+}: ListOfLinkTeasers): React.JSX.Element => {
+  if (isEmpty(linkTeasers)) {
     return null
   }
 
-  const renderLink = (uri: string, title: string, summary?: string) => (
+  const renderLink = (link: LinkTeaser) => (
     <>
       <p className="vads-u-margin--0">
         <va-link
           disable-analytics
           onClick={() =>
             recordEvent({
               event: 'nav-featured-content-link-click',
-              'featured-content-header': sectionTitle,
-              'featured-content-click-label': title,
+              'featured-content-header': title,
+              'featured-content-click-label': link.title,
             })
           }
-          href={uri}
-          text={title}
+          href={link.uri}
+          text={link.title}
         />
       </p>
-      {summary && <p className="vads-u-margin--0">{summary}</p>}
```

Comment:
[087e7f2](https://github.com/department-of-veterans-affairs/next-build/pull/1456/commits/087e7f2a77c5e95bfc041395984715ecbc827877)